llvm/lib/Target/ARM/AsmParser/ARMAsmParser.cpp

```diff
@@ -7118,6 +7161,11 @@ bool ARMAsmParser::ParseInstruction(ParseInstructionInfo &Info, StringRef Name,
     }
   }
 
+  // This marks the end of the LHS Mnemonic operators
```

Comment:
Statements should end with a period. Please make sure other comments are fixed as well.